Documentation/operations/troubleshooting_clustermesh.rst

```diff
@@ -7,25 +7,44 @@ Install the Cilium CLI
 
 .. include:: /installation/cli-download.rst
 
-Generic
--------
+Automatic Verification
+----------------------
 
- #. Validate that the ``cilium-xxx`` as well as the ``cilium-operator-xxx`` pods
-    are healthy and ready.
+ #. Validate that Cilium pods are healthy and ready:
 
     .. code-block:: shell-session
 
        cilium status
 
- #. Validate that Cluster Mesh is enabled correctly and operational:
+ #. Validate that Cluster Mesh is enabled and operational:
 
     .. code-block:: shell-session
 
        cilium clustermesh status
 
+ #. In case of errors, run the troubleshoot command to automatically investigate
+    Cilium agents connectivity issues towards the ClusterMesh control plane in
+    remote clusters:
+
+    .. code-block:: shell-session
+
+       kubectl exec -it -n kube-system ds/cilium -c cilium-agent -- cilium troubleshoot clustermesh
```

Comment:
Yes, because I only dropped it from the clustermesh-apiserver, while it is present as part of `cilium-dbg` (which seems to be still referenced as `cilium` in v1.14): https://github.com/cilium/cilium/pull/32571/commits/8a923c03cf33893504b8918733b4b044834f7c41